pyproject.toml

```diff
@@ -303,6 +303,12 @@ exclude = [
     "venv",
     '__init__.py',
     'src/zenml/cli/version.py',
+    # LitGPT files from the LLM Finetuning example
+    'examples/llm_finetuning/evaluate',
+    'examples/llm_finetuning/finetune',
+    'examples/llm_finetuning/generate',
+    'examples/llm_finetuning/lit_gpt',
+    'examples/llm_finetuning/scripts',
```

Comment:
Also not very confident here - why is this needed? You have linting in your `copier.yaml`, right? Why this is not aligned with what we have here?

Reply:
Same as above.